quantum/q_full_adder.py

```diff
@@ -10,13 +10,12 @@
 
 import math
 
-import qiskit
-from qiskit import Aer, ClassicalRegister, QuantumCircuit, QuantumRegister, execute
+import qiskit as q
```

Comment:
Using just `q` reduces readability for new users of qiskit and does not really save much space.
```suggestion
import qiskit
```

Reply:
Changes made